dali/python/nvidia/dali/types.py

```diff
@@ -518,22 +518,9 @@ def _convert(x, type):
     if device is None:
         device = "cpu"
 
-    # 'name' argument is pased to call, not the constructor
-    constructor_args = kwargs
-    call_args = {}
-    name = constructor_args.get("name")
-    if name is not None:
-        call_args["name"] = name
-        del constructor_args["name"]
-
-    op = ops.Constant(device=device,
-                      fdata=fdata,
-                      idata=idata,
-                      shape=shape,
-                      dtype=dtype,
-                      layout=layout,
-                      **constructor_args)
-    return op(**call_args)
+    op = fn.constant(device=device, fdata=fdata, idata=idata, shape=shape, dtype=dtype,
+                     layout=layout, **kwargs)
+    return op
```

Comment:
:+1: 
You can take it one step further:
```suggestion
    return fn.constant(device=device, fdata=fdata, idata=idata, shape=shape, dtype=dtype,
                       layout=layout, **kwargs)
```